misc_scripts/toaquality_query.py

```diff
@@ -23,7 +23,7 @@
 from tables import *
 from joins import *
 from graphql_client import GraphQLClient
-sys.path.append('/fred/oz005/users/acameron/pipeline_stuff/andrew_meerpipe_dev/meerpipe/')
```

Comment:
This path amendment is required to access the db_utils library. Happy to discuss ideas for a more effective way of implementing this, I agree it's not ideal.

Reply:
It is not ideal. It's currently not needed because the batch scripts `cd` to the directory of the software. In my nextflow branch I am working on making these scripts installed properly so they can be imported from any directory

Reply:
These scripts do not run as part of the pipeline. They are utility scripts designed for monitoring the behaviour of the pipeline and PSRDB, and at times for making specific changes. So I don't think the `cd` statement gets around that. 

One workaround might be to just drop a soft link (`ln -s ../db_utils.py`, etc.) in the relevant directories, which would save on the absolute filepath. You'd just possibly get a few extra .pyc files, I suppose.

Reply:
Resolved via PYTHONPATH modifications